games-vue-client/cypress/utils/constants.js

```diff
@@ -0,0 +1,2 @@
+export const BASE_URL = 'http://localhost:8080'
+export const GUEST_TOKEN = 'C3qH3VVmoqR/rbxJIiVys1jW5jE0bDtuixWVorDfJhE=';
```

Comment:
upps. Public now and 4-ever.

Reply:
The guest token can easily be changed later